### PR TITLE
Run installation in sandboxed environment 

### PR DESCRIPTION
**Pull request by dmarteau**
## Changes
* Remove useless Makefile
* Directory configuration run in sandboxed mode using docker container
* Update README for running installation

## Rational
This leverage the following:
- For people that run docker as sudoer it will prevent shooting themselves in the foot with misconfigured install dir (like erasing your system `/var` directory because `INSTALL_DEST` was not defined).
- Allow install as different user/group even if your are not sudoer but have docker execution rights.
  
The latter is the most important because it allows separation between admin of the docker stack
and the user/group executing services (usually the `lizmap` user).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,9 +38,9 @@ docker compose up
 
 ### Windows
 
-In order to user Docker on Windows you must install [Docker desktop for Windows](https://docs.docker.com/desktop/windows/install/)
```
Comment (Gustry) [inline README.md:41]: it's not required to install docker for windows ?

File: configure.sh [hunk 2]
```diff
@@ -3,25 +3,27 @@
 #
 # Check uid/gid of installation dir
 #
-
 set -e
 
+if [ -z $INSTALL_DEST ]; then
+# Define default install destination as currunt directory
```
Comment (Gustry) [inline configure.sh:9]:
```suggestion
# Define default install destination as current directory
```

**Review by Gustry — APPROVED** [review]
I haven't tried, I'm not the docker expert. LGTM

File: README.md (hunk 1 above)
Comment (dmarteau) [reply README.md:41]: Actually 'Docker desktop' is an option/suggestion not a requirement, there is other way to use docker on windows on top of WSL2.   Furthemore the subscription terms for Docker desktop has changed and I'm not sure that *requiring* is appropriate.

**Comment by nboisteault** [comment]
@nworr Could you test this PR in Windows please?

**Review by laurentj — APPROVED** [review]
LGTM

**Comment by nworr** [comment]
Got an error : 

`Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error mounting "/run/desktop/mnt/host/c/Users/rmartin/Documents/en cours/lizmap-docker-compose-secure-file-mode/lizmap/etc/nginx.conf" to rootfs at "/etc/nginx/nginx.conf": mount /run/desktop/mnt/host/c/Users/rmartin/Documents/en cours/lizmap-docker-compose-secure-file-mode/lizmap/etc/nginx.conf:/etc/nginx/nginx.conf (via /proc/self/fd/14), flags: 0x5000: not a directory: unknown: Are you trying to mount a directory onto a file (or vice-versa)? Check if the specified host path exists and is the expected type `

**Comment by nworr** [comment]
My folder got a space, the configure script was confused, but  when i a correct folder name, no more problem launching ngnix

**Comment by dmarteau** [comment]
@nboisteault @nworr is that ok for you to merge ?

**Review by nworr — APPROVED** [review]

**Review by nboisteault — APPROVED** [review]

**Comment by dmarteau** [comment]
@nworr  I have made some fixes, could you test one more time on Windows before merging ? thx.

**Comment by nworr** [comment]
still working, 
to clarify : i need wsl to launch `configure`

**Pull request merged**